### PR TITLE
LWDEV-8967: Update header

### PR DESCRIPTION
**Pull request by PaulRumyantsevItr**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -86,29 +57,10 @@ const Header: React.SFC<HeaderProps> = ({
                   <Icon name={'cogwheel'} color={colors.coolGrey} />
                 </span>
                 {settingsStore.showSettings ? (
-                  <SettingsModal handleCloseSettings={handleCloseSettings} />
+                  <SettingsModal handleCloseSettings={handleToggleSettings} />
```
Comment (sergeyschegolev) [inline src/components/Header/Header.tsx]: Think handleCloseSettings attribute should be renamed as well

Comment (PaulRumyantsevItr) [reply src/components/Header/Header.tsx]: Discussed

File: src/components/Terminal/Terminal.tsx [hunk 2]
```diff
@@ -231,26 +235,38 @@ class Terminal extends React.Component<TerminalProps, {}> {
...
-        )}
-        <Header history={this.props.history} />
-        <Mosaic
-          renderTile={this.handleRenderTile}
-          onChange={this.handleChange}
-          resize={{minimumPaneSizePercentage: MIN_PANE_SIZE_PERCENTAGE}}
-          initialValue={this.state.initialValue}
+        <Header
+          // tslint:disable-next-line:jsx-no-lambda
+          onLogout={() => this.onLogout()}
```
Comment (GeorgePuisha) [inline src/components/Terminal/Terminal.tsx]: Why not `this.onLogout`?

**Comment by GeorgePuisha** [comment]
By the way, resolve conflicts

File: src/components/Terminal/Terminal.tsx (hunk 2 above)
Comment (PaulRumyantsevItr) [reply src/components/Terminal/Terminal.tsx]: Because we are using this inside callback function

**Pull request merged**